test_that("Returns correct error message", {
  expect_equal(2 * 2, 4)
})
